cmd/clear.go

```diff
@@ -8,43 +8,51 @@ import (
 func clearCommand() *cobra.Command {
 	use, aliases := generateShellAlias("clear")
 	clearCmd := &cobra.Command{
-		Use:     use + " [<path>]",
+		Use:     use + " [<path>]...",
 		Aliases: aliases,
-		Short:   "Clears the cache at <path>, or current directory if not specified",
+		Short:   "Clears the cache at the specified paths, or current directory if not specified",
 		Long: `Wash caches most operations. If the resource you're querying appears out-of-date, use this
-subcommand to reset the cache for resources at or contained within <path>. Defaults to the current
-directory if <path> is not specified.`,
-		Args: cobra.MaximumNArgs(1),
+subcommand to reset the cache for resources at or contained within the specified paths.
+Defaults to the current directory if no path is provided.`,
 		RunE: toRunE(clearMain),
 	}
 	clearCmd.Flags().BoolP("verbose", "v", false, "Print paths that were cleared from the cache")
 	return clearCmd
 }
 
 func clearMain(cmd *cobra.Command, args []string) exitCode {
-	path := "."
+	paths := []string{"."}
 	if len(args) > 0 {
-		path = args[0]
+		paths = args
 	}
 	verbose, err := cmd.Flags().GetBool("verbose")
 	if err != nil {
 		panic(err.Error())
 	}
 
 	conn := cmdutil.NewClient()
-	cleared, err := conn.Clear(path)
-	if err != nil {
-		cmdutil.ErrPrintf("%v\n", err)
-		return exitCode{1}
-	}
 
-	if verbose {
-		for _, p := range cleared {
-			cmdutil.Println("Cleared", p)
+	// Perform the operation. Note that wclear isn't parallelized because
```

Comment:
Is there some reason you didn't parallelize the rest?

Reply:
Yeah, because we need to fetch all the data before doing anything else. But I can go ahead and parallelize that part, it's not that much harder and it is a bit of a performance gain.

Reply:
I meant, didn't parallelize the other commands in that commit.

Reply:
Yeah, that's what I'm referring to.